tools/package-tools/gulp/build-tasks-gulp.ts

```diff
@@ -136,7 +136,9 @@ export function createPackageBuildTasks(buildPackage: BuildPackage, preBuildTask
         .pipe(dest(buildPackage.esm5OutputDir));
   });
 
-  task(`${taskName}:assets:inline`, () => inlineResourcesForDirectory(buildPackage.outputDir));
+  task(`${taskName}:assets:inline`, () => {
+    return inlineResourcesForDirectory(buildPackage.outputDir);
+  });
```

Comment:
What's the change here?

Reply:
I decided to put it on a new line with an explicit `return` so that we don't break it in the same way as the one above, if we ever decide to add more logic to the function. My guess is that's what happened with the first one.